homeassistant/components/yolink/sensor.py

```diff
@@ -0,0 +1,23 @@
+"""Support for Yolink Devices."""
+
+from homeassistant.components.yolink.device_impl import YoLinkDoorSensor
+
+from .const import DOMAIN, HOME_SUBSCRIPTION
+from .device import YoLinkDevice
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities):
+    """Perform the setup for YoLink devices."""
+    devices = []
+    entities: list[YoLinkDevice] = []
+    for device in hass.data[DOMAIN][config_entry.entry_id]["devices"]:
+        if device["type"] == "DoorSensor":
+            devices.append(YoLinkDoorSensor(device, hass, config_entry))
```

Comment:
Can you not directly add the YoLinkDoorSensor to `entities`?

Reply:
There will be many different sensor under the user account. I want to integrate the door sensor first